internal/config/config_file.go

```diff
@@ -30,6 +31,34 @@ func ParseDefaultConfig() (Config, error) {
 	return ParseConfig(ConfigFile())
 }
 
+// Get the absolute path for a folder or file. Support for the old homedir method
+// Tied to https://github.com/cli/cli/pull/2740
+func GetAbsolutePath(elem ...string) (string, error) {
+	oldHomeDir, err := homedir.Dir()
+	if err != nil {
+		return "", err
+	}
+
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return "", err
+	}
+
+	fullOldPath := filepath.Join(append([]string{oldHomeDir}, elem...)...)
+	fullPath := filepath.Join(append([]string{homeDir}, elem...)...)
+
+	if fullOldPath != fullPath {
```

Comment:
Could we avoid renaming if `elem` was empty? This is to have a guard against ever renaming the literal user's home directory, which of course we never want to move/rename.